src/server/tests/server_test.go

```diff
@@ -132,3 +132,134 @@ func TestProtocols(t *testing.T) {
 		test_utils.AssertEqual(t, got, expected)
 	})
 }
+
+func TestYodelHandlers(t *testing.T) {
```

Comment:
Needs test to validate yodel ID.

Reply:
I believe the `yodel creation sends back id` test covers that.

Reply:
Doesn't the ID need to be a valid hex, or are we accepting arbitrary IDs? On line 203 there's a test which suggests that only valid hex strings can be used as IDs.